package/MDAnalysis/analysis/encore/confdistmatrix.py

```diff
@@ -0,0 +1,435 @@
+stored in an Ensemble. A class to compute an RMSD matrix in such a way is also
+available.
+
+:Author: Matteo Tiberti, Wouter Boomsma, Tone Bengtsen
+:Year: 2015--2016
+:Copyright: GNU Public License v3
+:Mantainer: Matteo Tiberti <matteo.tiberti@gmail.com>, mtiberti on github
+
+.. versionadded:: 0.14.0
+
+"""
+
+from multiprocessing import Process, Array, cpu_count, RawValue
+from numpy import (sum, average, transpose, dot, ones, asarray, mean,
+                   float64, object, bool, array, int)
+from ctypes import c_float
+from getpass import getuser
+from socket import gethostname
+from datetime import datetime
+from time import sleep
+
+from ..align import rotation_matrix
+
+from .cutils import PureRMSD, MinusRMSD
+from .utils import TriangularMatrix, trm_indeces, AnimatedProgressBar
+
+
+class ConformationalDistanceMatrixGenerator(object):
+    """
+    Base class for conformational distance matrices generator between array of
+    coordinates. Work for single matrix elements is performed by the private
+    _simple_worker and _fitter_worker methods, which respectively do or don't
+    perform pairwise alignment before calculating the distance metric. The
+    class efficiently and automatically spans work over a prescribed number of
+    cores, while keeping both input coordinates and the output matrix as
+    shared memory. If logging level is low enough, a progress bar of the whole
+    process is printed out. This class acts as a functor.
+    """
+
+    def run(self, ensemble, selection="", superimposition_selection="",
+            ncores=None, pairwise_align=False, mass_weighted=True,
+            metadata=True):
+        """
+        Run the conformational distance matrix calculation.
+
+        Parameters
+        ----------
+
+        ensemble : encore.Ensemble.Ensemble object
+            Ensemble object for which the conformational distance matrix will
+            be computed.
+
+        pairwise_align : bool
+            Whether to perform pairwise alignment between conformations
+
+        align_subset_coordinates : numpy.array or None
+            Use these coordinates for superimposition instead of those from
+            ensemble.superimposition_coordinates
+
+        mass_weighted : bool
+            Whether to perform mass-weighted superimposition and metric
+            calculation
+
+        metadata : bool
+            Whether to build a metadata dataset for the calculated matrix
+
+        ncores : int
+            Number of cores to be used for parallel calculation
+
+        Returns
+        -------
+
+        conf_dist_matrix` : encore.utils.TriangularMatrix object
+            Conformational distance matrix in triangular representation.
+
+        """
+
+        # Decide how many cores have to be used. Since the main process is
+        # stopped while the workers do their job, ncores workers will be
+        # spawned.
+        if not ncores:
+            ncores = cpu_count()
```

Comment:
I think you should default to 1 core. People might be running other things as well or don't have enough memory to feed all processes/threads started by this.